Build/Tactical/TeamTurns.cc

```diff
@@ -146,6 +148,9 @@ void StartPlayerTeamTurn( BOOLEAN fDoBattleSnd, BOOLEAN fEnteringCombatMode )
   /// that a baddie can die and still maintain it's attacker ID
   gTacticalStatus.fKilledEnemyOnAttack = FALSE;
 
+  // Save if we are in Dead is Dead mode
+  if (gGameOptions.ubGameSaveMode == DIF_DEAD_IS_DEAD)   DoDeadIsDeadSave();
```

Comment:
Up for debate: would it be cleaner to inline the check into the function? One included header less, perhaps in some other file too.